src/pages/index.js

```diff
@@ -98,4 +99,30 @@ const Index = () => {
   );
 };
 
+export const getStaticProps = async () => {
+  const fallbackUserCount = 200000;
+
+  try {
+    const r = await fetch(
+      `https://discord.com/api/v9/invites/reactiflux?with_counts=true&with_expiration=true`,
+      { method: "get" },
+    );
+    const data = await r.json();
+
+    return {
+      revalidate: 60,
```

Comment:
Apologies for being a massive pedant again but is there any value in variablising the revalidate time?

Reply:
This is the only page that uses ISR; there wouldn't by any benefit rn

Reply:
I see, that makes sense 👍